lua/telescope/_extensions/zoxide/list.lua

```diff
@@ -103,8 +103,11 @@ return function(opts)
 
   local z_config = require("telescope._extensions.zoxide.config")
   local cmd = z_config.get_config().list_command
-
-  opts.cmd = utils.get_default(opts.cmd, {vim.o.shell, "-c", cmd})
+  local shell_arg = "-c"
+  if vim.o.shell == "cmd.exe" then
```

Comment:
Should and can we generalise this for other Windows terminals. Like, does this work for Powershell? I am not really familiar with Windows shells anymore and does not have a test machine for it.

Reply:
I just tested powershell7,nyagos,evlish,nushell with "/c" in Windows, they work fine, only old cmd.exe and yori.exe needs "-c" flag.

Reply:
Ty for testing!